package.yaml

```
@@ -0,0 +1,4 @@
+dependencies:
+  location:
+    url: github.com/toitware/toit-location
+    version: 1.0.0
```

You don't want to depend on a single version.
```suggestion
    version: ^1.0.0
```